public/content/lessons/2017/derivatives-trig-functions/index.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: Trig Derivatives through geometry
+description: Introduction to the derivatives trigonometric functions thought about geometrically and intuitively.
+date: 2017-04-30
+chapter: 4
+video: S0_qX4VJhMQ
+timestamp: 756
+source: _2017/eoc/chapter3.py
+credits:
+- Lesson by Grant Sanderson
+- Text adaptation by Kurt Bruns
+---
+
+Let’s tackle one more type of function: Trigonometric functions. In particular, let’s focus on the sine function. Right here I’m going to assume you are familiar with how to think about trigonometric functions using the unit circle; the circle with radius $1$ centered at the origin.
+
+For a given value of theta, like $0.8$, you’d imagine walking around the circle, starting from the rightmost point, until you’ve traversed that distance $0.8$ in arc length. This is the same thing as saying that this angle here is exactly $0.8$ radians, since the circle has radius $1$. Then $\sin(\theta)$ is your height above the $x$-axis at this point.
```

Comment:
```suggestion
For example, how would you interpret the value $\sin(0.8)$ if the value $\theta = 0.8$ is understood to be in radians?
You might imagine walking around a circle with a radius of $1$, starting from the rightmost point, until you’ve traversed the distance $0.8$ in arc length.
This is the same thing as saying you've traversed an _angle_ of $0.8$ radians. Then $\sin(\theta)$ is your height above the $x$-axis at this point.
```

Reply:
When the script includes phrases like "this angle here", that's usually with the intent that it's spoken while a specific thing is referenced on screen.  For the articles, sentences including those kinds of phrases are worth rewriting.